src/data-validation.js

```diff
@@ -167,10 +167,15 @@ const addons_flyout = {
       properties: {
         current: {
           type: "object",
-          required: ["slug", "single_version"],
+          required: ["slug", "versioning_schema"],
           properties: {
             slug: { type: "string" },
-            single_version: { type: "boolean" },
+            versioning_schema: {
```

Comment:
Same for all other places

```suggestion
            versioning_scheme: {
```